src/SDK/Language/HTTP.php

```diff
@@ -172,12 +173,12 @@ public function getParamExample(array $param)
     public function getFiles()
     {
         return [
-            [
-                'scope'         => 'method',
-                'destination'   => 'docs/examples/{{service.name | caseLower}}/{{method.name | caseDash}}.md',
-                'template'      => 'http/docs/example.md.twig',
-                'minify'        => false,
-            ],
+          [
+            'scope'         => 'method',
+            'destination'   => 'docs/examples/{{service.name | caseLower}}/{{method.name | caseDash}}',
+            'template'      => '/http/docs/service.twig',
```

Comment:
example.md.twig will be more consistent use of the file name compared to other language templates.